qa/L0_analyze/test.sh

```diff
@@ -34,7 +34,7 @@ TRITON_SERVER_VERSION="21.02-py3"
 CLIENT_PROTOCOL="grpc"
 PORTS=(`find_available_ports 3`)
 GPUS=(`get_all_gpus_uuids`)
-OUTPUT_MODEL_REPOSITORY=${OUTPUT_MODEL_REPOSITORY:='/tmp/output/model_repository'}
```

Comment:
The reason I added this was if we wanted to manually specify an output model repository at the command line during a local test.  Could we make it 
```
OUTPUT_MODEL_REPOSITORY
=${OUTPUT_MODEL_REPOSITORY:=(`get_output_directory`)}
```

Reply:
I see. Sorry. It's fixed now.